src/components/Table/components/Row.tsx

```diff
@@ -113,7 +120,7 @@ function RowImpl<R extends Kinded, S>(props: RowProps<R>): ReactElement {
   let expandColumnHidden = false;
 
   return (
-    <RowTag css={rowCss} {...others} data-gridrow {...getCount(row.id)}>
+    <RowTag css={rowCss} {...others} data-gridrow {...getCount(row.id)} draggable={row.draggable}>
```

Comment:
**thought/suggestion:** I think it'd be great to make a specific "drag handle" and test out what that looks like as well. 
If we have a row with many other interactive elements inside of it we could have some potentially conflicting interactions (i dunno if that'd be true, but I'd rather avoid it).

Reply:
Yeah I'm planning on adding that as well. Specifically on mobile I think it is necessary since dragging is the default behavior for scrolling through the table.